### ISSUE TITLE
`delete day` deletes the wrong day

### ISSUE THREAD

**Issue by Kyzure** [Kyzure/pe#4]
Before:
![image.png](https://raw.githubusercontent.com/Kyzure/pe/master/files/1c903f27-6c6e-499c-875b-b862afb1e626.png)

After:

![image.png](https://raw.githubusercontent.com/Kyzure/pe/master/files/fa177311-1d35-450f-bfd6-89d66c75a3ea.png)

It does not delete the first day, but the last day.

Otherwise, deleting from the back:

After:

![image.png](https://raw.githubusercontent.com/Kyzure/pe/master/files/90e54670-31e2-4a67-8b12-ae4d5a848617.png)

This also deletes the last day, and not any other day.

Basically, delete day always deletes the last day no matter the index given.
